docs/src/main/asciidoc/logging.adoc

```diff
@@ -789,23 +797,22 @@ You get messages containing the MDC data:
 Depending on the API you use, the MDC class is slightly different.
 However, the APIs are very similar:
 
-* log4j 1 - `org.apache.log4j.MDC.put(key, value)`
-* log4j 2 - `org.apache.logging.log4j.ThreadContext.put(key, value)`
-* slf4j - `org.slf4j.MDC.put(key, value)`
+* Log4j 1 - `org.apache.log4j.MDC.put(key, value)`
+* Log4j 2 - `org.apache.logging.log4j.ThreadContext.put(key, value)`
+* SLf4J - `org.slf4j.MDC.put(key, value)`
 
 [[mdc-propagation]]
 ==== MDC propagation
 
-Under the hood, Quarkus provides a specific implementation of the MDC provider handling the reactive context.
-Thus, the MDC data is propagated even when using reactive and asynchronous processing.
+In Quarkus, the MDC provider has a specific implementation for handling the reactive context, ensuring that MDC data is propagated during reactive and asynchronous processing.
 
-Consequently, the MDC data is still available:
+As a result, you can still access the MDC data in various scenarios:
 
-- after async calls (like a REST client returning a Uni)
-- in the code submitted to the `ManagedExecutor` (`@Inject org.eclipse.microprofile.context.ManagedExecutor executor`)
-- in the code executed using `vertx.executeBlocking()`
+* After asynchronous calls, like for example, when a REST client returns a Uni.
+* In code submitted to `org.eclipse.microprofile.context.ManagedExecutor`.
+* In code executed with `vertx.executeBlocking()`.
 
-NOTE: When available, the MDC data is stored on a _duplicated context_ which is an isolated context for your processing.
+NOTE: If possible, MDC data is stored in a _duplicated context_, which is an isolated context for processing a single task (request).
```

Comment:
This doesn't seem right. Should this be an instruction?

```suggestion
NOTE: Where possible, store MDC data in a _duplicated context_, which is an isolated context for processing a single task (request).
```
Or is this a clarification:

```suggestion
NOTE: In most instances, MDC data is stored in a _duplicated context_, which is an isolated context for processing a single task (request).
```
Or other?

Reply:
This is _not_ an instruction. A duplicated context does not always exist, but when it is, Quarkus makes sure the MDC data are stored in it.

Reply:
@Ladicek - I get it now. Thanks :+1: 
```suggestion
NOTE: If applicable, MDC data is stored in a _duplicated context_, which is an isolated context for processing a single task (request).
```